bucket/nomeiryoui.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "2.41",
+    "description": "Windows system font setting tool on Windows 8.1/10.",
+    "homepage": "https://github.com/Tatsu-syo/noMeiryoUI",
+    "license": "MIT",
+    "url": "https://github.com/Tatsu-syo/noMeiryoUI/releases/download/TAG-2.41/noMeiryoUI241.zip",
+    "hash": "9acb61ab3c9e1d6ba1f0823ba777d4c418fbc0b67a04588442082317868c0293",
+    "bin": "noMeiryoUI.exe",
+    "shortcuts": [
+        [
+            "noMeiryoUI.exe",
+            "noMeiryoUI"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Tatsu-syo/noMeiryoUI"
+    },
```

```suggestion
    "checkver": "github",
```
https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#special-cases